### DISCUSSION TITLE
presigned url for external_image_storage.s3

### DISCUSSION THREAD

**Discussion by tparsa** [grafana/grafana#66284] [Alerting]
Hello,
We have a signed URL option with GCD but no option to have resigned URLs in general S3 image storage. Some panels might have delicate data that we don't want publicly visible.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82707. Feel free to continue the discussion around this there. Thank you!
